apps/dotcdn/src/app/app.component.html

```diff
@@ -0,0 +1,88 @@
+<p-tabView>
+    <p-tabPanel header="Overview" *ngIf="vm$ | async as VM">
+        <div class="dot-cdn__tab-content">
+            <p-dropdown
+                [options]="periodValues"
+                (onChange)="changePeriod($event)"
+                optionLabel="label"
+                optionValue="value"
+                [(ngModel)]="selectedPeriod.value"
+            ></p-dropdown>
+            <div class="dot-cdn__stats">
+                <div class="dot-cdn__col" *ngFor="let stats of VM.statsData">
+                    <span class="dot-cdn__stats-label">{{ stats.label }}</span>
+                    <p-skeleton
+                        styleClass="skeleton"
+                        width="10rem"
+                        height="1.5rem"
+                        *ngIf="VM.isChartLoading; else valuesBlock"
+                    ></p-skeleton>
+                    <ng-template #valuesBlock>
+                        <h3 class="dot-cdn__stats-value">{{ stats.value }}</h3>
+                    </ng-template>
+                    <dot-icon [name]="stats.icon" size="32"></dot-icon>
+                </div>
+            </div>
+
+            <div class="dot-cdn__chart">
+                <dot-spinner
+                    [size]="'4rem'"
+                    [borderSize]="'3px'"
+                    *ngIf="VM.isChartLoading; else chartBlock"
+                    [ngStyle]="{ height: chartHeight }"
+                ></dot-spinner>
+                <ng-template #chartBlock>
+                    <h3 class="dot-cdn__tab-content-label">Bandwidth Served</h3>
+                    <p-chart
+                        #chart
+                        type="line"
+                        [data]="VM.chartData"
+                        width="100%"
+                        [height]="chartHeight"
+                        [options]="options"
+                    ></p-chart>
+                </ng-template>
+            </div>
+        </div>
+    </p-tabPanel>
+    <p-tabPanel header="Flush Cache" *ngIf="vmPurgeLoaders$ | async as VMPurgeLoaders">
+        <div class="dot-cdn__tab-content--contained">
+            <form [formGroup]="purgeZoneForm">
+                <div class="dot-cdn__tab-content__row">
+                    <h3 class="dot-cdn__tab-content-label">Purge URL List</h3>
+                    <p>
+                        Purging an url list will remove the file from the CDN cache and re-download
+                        it from your server. Please enter the exact CDN URL of each individual file.
+                        Wildcard values are not supported.
+                    </p>
+                    <textarea
+                        [rows]="5"
+                        pInputTextarea
+                        formControlName="purgeUrlsTextArea"
+                    ></textarea>
+                    <button
```

Comment:
Let's add a hint here that reads:

"Purging an URL list will remove the file from the CDN cache and re-download it from your server. Please enter the exact CDN URL of each file. You can also purge folders or wildcard files using * inside of the URL path."

Reply:
Actually, let's not use the verbiage "Pull Zone" at all as it is specific for Bunny and is not general nomenclature.  Let's just change the button to "Purge All".

```
Purge All

"Purge All" will remove files from your CDN domain and force everything to be re-downloaded 
from your origin site. If your CDN domain is highly trafficked and/or caches a lot of resources, 
this might send a large amount of traffic back to your origin sites, so use with caution.
```